backend/repositories/match-repository.js

```diff
@@ -101,6 +101,13 @@ exports.recordMatch = async (matchDescription) => {
     const match = await constructMatch(matchDescription)
     const result = await storeMatch(match)
     const newUsers = await storage.getAllUsers()
-    bot.reportMatchOnSlack(match, oldUsers, newUsers)
+
+    try {
+        const bot = await botFactory.makeBot(process.env.FOOSBOT_TOKEN, process.env.FOOS_CHANNEL_NAME)
```

Comment:
Please don't initialize the bot over again with every match.

Reply:
good point, thanks